locale/de/LC_MESSAGES/django.po

```diff
@@ -1054,75 +1038,82 @@ msgstr "Entfernen"
 msgid "Edit project"
 msgstr "Projekt bearbeiten"
 
-#: apps/dashboard/views.py:168
+#: apps/dashboard/views.py:167
 msgid "Organisation successfully updated."
 msgstr "Organisation erfolgreich aktualisiert"
 
-#: apps/documents/phases.py:16
-msgid "Create document phase"
-msgstr "Dokument-Anlegen Phase"
+#: apps/documents/phases.py:17
+msgid "Comment phase"
+msgstr "Kommentierphase"
 
-#: apps/documents/phases.py:17 apps/documents/phases.py:30
+#: apps/documents/phases.py:18
 msgid "commenting text"
 msgstr "Text kommentieren"
 
-#: apps/documents/phases.py:18
-msgid "Create text for the project."
-msgstr "Einen Text für das Projekt anlegen."
-
-#: apps/documents/phases.py:29
-msgid "Comment phase"
-msgstr "Kommentierphase"
-
-#: apps/documents/phases.py:31
+#: apps/documents/phases.py:19
 msgid "Collect comments for the text."
 msgstr "Kommentieren von Text und einzelnen Absätzen."
 
-#: apps/documents/templates/meinberlin_documents/document_detail.html.py:21
-msgid "Comments for this Paragraph"
-msgstr "Kommentare für diesen Absatz"
+#: apps/documents/templates/meinberlin_documents/chapter_detail.html.py:16
+#: apps/documents/templates/meinberlin_documents/chapter_detail.html.py:23
+msgid "Table of Contents"
+msgstr "Inhaltsverzeichnis"
 
-#: apps/documents/templates/meinberlin_documents/document_form.html.py:13
-msgid "Document is being created."
-msgstr "Dieses Dokument wird noch angelegt."
+#: apps/documents/templates/meinberlin_documents/chapter_detail.html.py:57
+#, python-format
+msgid "%(count)s Comment"
+msgid_plural "%(count)s Comments"
+msgstr[0] "%(count)s Kommentar"
+msgstr[1] "%(count)s Kommentare"
 
-#: apps/documents/templates/meinberlin_documents/paragraph_detail.html.py:14
-msgid "back to document"
-msgstr "Zurück zum Dokument"
+#: apps/documents/templates/meinberlin_documents/document_management.html.py:15
+#: apps/polls/templates/meinberlin_polls/poll_management_form.html.py:15
+msgid "Manage project contents"
+msgstr "Projektinhalt verwalten"
+
+#: apps/documents/templates/meinberlin_documents/includes/nextprev.html.py:9
+#: apps/documents/templates/meinberlin_documents/includes/nextprev.html.py:14
+msgid "Previous Chapter"
+msgstr "Letztes Kapitel"
```

Comment:
"Letztes" can mean both "previous" and "last". Unfortunately, "vorherig" is not much better.